src/commoncode/filetype.py

```diff
@@ -45,20 +45,24 @@ def is_link(location):
     return location and os.path.islink(location)
 
 
-def is_file(location):
+def is_file(location, allow_symlinks=False):
```

Comment:
What about using the same name as in the standard lib?
https://docs.python.org/3/library/os.html?highlight=link#os.link
https://docs.python.org/3/library/os.html?highlight=link#follow-symlinks

> If `follow_symlinks` is False, and the last element of the path to operate on is a symbolic link, the function will operate on the symbolic link itself rather than the file pointed to by the link. (For POSIX systems, Python will call the l... variant of the function.)

Reply:
That's a good idea, I'll change it.